src/main/java/frc/robot/constants/SwerveConstants.java

```diff
@@ -1,9 +1,9 @@
 package frc.robot.constants;
 
 public final class SwerveConstants {
-    public static final double MAX_SPEED = 5.0; // m/s
-    public static final double MAX_ANGULAR_VELOCITY = 1.0; // rad/s
+    public static final double MAX_SPEED = 2; // m/s
```

Comment:
This seems a little slow, I trust that it's for good reason?

Reply:
I've probably been doing something wrong, but YAGSL seems to not care about the max speed. The drive seems open loop even though it's in closed loop mode 

I'm sure if we had more time / actually needed this we coild figure it out